src/styles/header.scss

```diff
@@ -0,0 +1,113 @@
+header {
+    position: fixed;
+    z-index: 999;
+    width: 100%;
+    height: 3rem;
+    min-height: 50px;
+    color: #fff;
+    background: rgba(0, 8, 81, 0.8);
+    border: 1px solid transparent;
+    border-radius: 0;
+
+    .input-group {
+      float: right;
+      width: 200px;
+      margin: 0.7rem 0 0;
+
+      @media (max-width: 720px) {
```

Comment:
Refactoring the media query breakpoints and other constants into a centralized design tokens file is a crucial step toward enhancing our React codebase's maintainability and scalability. This consolidation fosters consistency throughout the UI, making it easier to manage and update styles across various components.